### PR TITLE
COM-349 Patient registration details change requests

### PR DESCRIPTION
**Pull request by prashantsable1991**
Added changes for patient registration details change requests.

### CODE REVIEW

File: openmrs/i18n/registration/locale_fr.json [hunk 1]
```diff
@@ -27,7 +27,9 @@
   "REGISTRATION_INSTITUTE_NAME_TEXT_KEY": "MINSANTE",
   "REGISTRATION_INSTITUTE_REGISTERED_TEXT_KEY": "(Enregistré)",
   "REGISTRATION_INSTITUTE_ADDRESS": "Hopital de la Cité Verte, District - Yaoundé",
-  "REGISTRATION_EMERGENCY_CONTACT_KEY": "No. Tel Personne à prévenir",
+  "REGISTRATION_EMERGENCY_CONTACT_NAME_KEY": "Emergency contact name",
```
Comment (lauravignoli) [inline openmrs/i18n/registration/locale_fr.json]: @prashantsable1991 Can you please get this translations from Lisa?

File: openmrs/i18n/registration/locale_fr.json [hunk 2]
```diff
@@ -53,11 +55,12 @@
   "ADDRESSHIERARCHY_CITY":"Ville",
   "ADDRESSHIERARCHY_DISTRICT":"District",
   "REGISTRATION_IDTYPE_1_CNI_KEY":"Numéro CNI",
-  "REGISTRATION_IDTYPE_2_ART_KEY":"Numéro TARV",
-  "REGISTRATION_IDTYPE_3_ANC_KEY":"Numéro CPN",
-  "REGISTRATION_IDTYPE_4_PASSPORT_KEY":"Numéro de Passport",
-  "REGISTRATION_IDTYPE_5_RECEIPT_KEY":"Numéro de Récépissé",
-  "REGISTRATION_IDTYPE_6_OTHER_KEY":"Autre numéro",
+  "REGISTRATION_IDTYPE_2_UIN_KEY": "Numero unique CNI",
```
Comment (lauravignoli) [inline openmrs/i18n/registration/locale_fr.json]: @prashantsable1991 Please use "Numéro unique CNI" here

**Comment by lauravignoli** [comment]
@prashantsable1991 Great work!!! I left minor comments. Also, I'm not sure why but the line in Autre Information disappeared, do you also get it?

autreInfoNoLine

File: openmrs/i18n/registration/locale_fr.json (hunk 1 above)
Comment (prashantsable1991) [reply openmrs/i18n/registration/locale_fr.json]: @lauravignoli I already ask Lisa about missing translations but she did not reply for that.

File: openmrs/i18n/registration/locale_fr.json (hunk 2 above)
Comment (prashantsable1991) [reply openmrs/i18n/registration/locale_fr.json]: @lauravignoli on ticket as translations provided to me I added that. I think we need to confirm all translations are added and they are correct as well before adding a ticket in "Ready for dev". I'm adding a translation provided by you. Thanks.

**Comment by prashantsable1991** [comment]
> @prashantsable1991 Great work!!! I left minor comments. Also, I'm not sure why but the line in Autre Information disappeared, do you also get it?
> 
> autreInfoNoLine

It's working fine:
COM-349

**Pull request merged**